Stylesheets/Base/typography.css

```diff
@@ -0,0 +1,6 @@
+@import url("https://fonts.googleapis.com/css2?family=Heebo:wght@200;300;400;500;600;800;900&family=Rubik:wght@300;400;600;700&display=swap");
+
+body {
+  font-family: "Heebo", sans-serif;
+  font-family: "Rubik", sans-serif;
```

Comment:
Only one of these two gets applied. Choose one.

Reply:
okay. i ll take one.